### ISSUE TITLE
Mary at Dabra Mǝtmāq

### ISSUE THREAD

**Issue by GueshSol** [BetaMasaheft/Documentation#2262]
In the MS AIC 2002.4 a miniature showing Saint Mary surrounded by a group of men playing traditional music for her glorification.
May I create a new art theme record for this miniature?
Thank you for your help.

**Comment by gnizla** [comment]
Hello, ideally take these the following steps: 1. identify if the scene is an illustration to a particular Miracle of Mary in works; 2. if it does illustrate a Work then ideally mention the Clavis ID of the text it illustrates; 3. Make sure no one has created an Art Theme for this subject, and if no one has you create a new AT.

**Comment by eu-genia** [comment]
I do not think this has ever been created, @GueshSol do you need it? I guess it is the one on f. 54r

**Comment by eu-genia** [comment]
@GueshSol please also check all issues you have opened https://github.com/BetaMasaheft/Documentation/issues?q=is%3Aissue%20state%3Aopen%20author%3AGueshSol to see if any action from you or anyone else is needed in order to be able to close them

**Comment by GueshSol** [comment]
@eu-genia thank you for the reminder. I will create one for Mary at Dabra Mǝtmāq (#2262) art theme and ʾO-ʾƎgzǝʾǝtǝya Māryām stanzas (#2985)  today, but issues #2405 , #2908  and #2908  are either fixed or no longer important; therefore will close them now.
